docs/README.rst

```diff
@@ -1,13 +1,13 @@
 How to build the docs
 =====================
 
-From the root directory of the repository, do the following to steps
+From the root directory of the repository, do the following in steps
 
 1. Install docs' dependencies
 
    .. code-block:: text
 
-       pip install -r docs/requirements.txt
+       pip install --group 'docs'
 
    On Linux, you may need to use `pip3` instead.
```

Comment:
On Linux, users are no longer allowed to invoke `pip` system-wide. They have to use a venv. So, this line can be removed.

```suggestion
```